client/lib/products-values/index.js

```diff
@@ -202,15 +202,25 @@ export function isJetpackMonthlyPlan( product ) {
 	return isMonthly( product ) && isJetpackPlan( product );
 }
 
-export function isJetpackProduct( product ) {
-	const jetpackProducts = [ ...JETPACK_BACKUP_PRODUCTS ];
+export function isJetpackProduct( product, type_of_product = null ) {
```

Comment:
I don't understand what's going on here. This isn't very typical, we try to keep these functions as simple as possible and with less parameters as possible.

I still believe that considering that we have only 1 product right now, this is over-engineering. We have other places in the code that only support this one product, too. 

YAGNI?